src/web/jest.config.js

```diff
@@ -10,11 +15,11 @@ module.exports = {
   collectCoverageFrom: ['src/**/*{ts,tsx}'],
   transform: {
     '^.+\\.(ts)$': 'ts-jest',
-    '^.+\\.(tsx)$': '<rootDir>/node_modules/babel-jest',
+    '^.+\\.(tsx)$': ['babel-jest', { presets: ['next/babel'] }],
```

Comment:
So we still have to define `babel-jest` here even though you are removing?  Can you clarify for me so I understand?